### PR TITLE
portal traceback

### PR DESCRIPTION
**Pull request by jpr-odoo**
Description of the issue/feature this PR addresses:
Task: https://www.odoo.com/web#id=36620&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/controllers/portal.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def _prepare_portal_layout_values(self):
         ])
 
         values['invoice_count'] = invoice_count
+        values['is_documents'] = bool(invoice_count)
```
Comment (tivisse) [inline addons/account/controllers/portal.py]: Remove the is_documents key.

Comment (tivisse) [inline addons/account/controllers/portal.py]: Remove the is_documents key.

Comment (jpr-odoo) [reply addons/account/controllers/portal.py]: why ? show message, if there is no documents into the account

Comment (tde-banana-odoo) [reply addons/account/controllers/portal.py]: Cfr task. We won't show a message to tell there is nothing to show. Keep portal simple.

**Comment by tde-banana-odoo** [comment]
Canceled

**Pull request closed without merging**